lib/cc/analyzer/container.rb

```diff
@@ -116,7 +154,11 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
```

Comment:
I'm not sure on the behavior of this snippet:

```
CONTAINER_TIMEOUT_SECONDS=
```

But the behavior would be the same either way. What does a `0` timeout or max bytes mean behavior-wise?